docs/classes/default.md

```diff
@@ -0,0 +1,1281 @@
+[toucan-sdk](../README.md) / [Exports](../modules.md) / default
```

Comment:
Should I review this entire file as well?

Reply:
nope